tests/ui/test_pvc_ui.py

```diff
@@ -20,25 +21,97 @@ def teardown(self):
         pvcs = [pvc_obj for pvc_obj in pvc_objs if "test-pvc" in pvc_obj.name]
         delete_pvcs(pvc_objs=pvcs)
 
-    @ui
+    @tier1
     @skipif_ocs_version("<4.6")
     @pytest.mark.parametrize(
-        argnames=["sc_type", "pvc_name", "access_mode", "pvc_size"],
+        argnames=["sc_type", "pvc_name", "access_mode", "pvc_size", "vol_mode"],
         argvalues=[
             pytest.param(
-                *["ocs-storagecluster-cephfs", "test-pvc-fs", "ReadWriteMany", "2"]
-            )
+                *[
+                    "ocs-storagecluster-cephfs",
+                    "test-pvc-fs",
+                    "ReadWriteMany",
+                    "2",
+                    "Filesystem",
+                ]
```

Comment:
```suggestion
                 "ocs-storagecluster-cephfs",
                 "test-pvc-fs",
                 "ReadWriteMany",
                 "2",
                 "Filesystem",
```
Should not this be written only like this? Instead of expanding list which is one extra step here? It also using some space in memory when you define this list which is not used anywhere else.

Or what's the point here?

Reply:
valid point, changed as suggested